### PR TITLE
Use git submodules

### PR DESCRIPTION
**Pull request by mjethani**
This PR updates the code to use https://github.com/gorhill/ublock and https://github.com/adblockplus/adblockpluscore as [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules).

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[submodule "packages/adblocker-benchmarks/blockers/ublock"]
```
Comment (mjethani) [inline .gitmodules:1]: Note that uAssets is now a submodule of uBlock.

File: packages/adblocker-benchmarks/Makefile [hunk 2]
```diff
@@ -4,21 +4,19 @@ all: deps run
 requests.json:
 	curl https://cdn.cliqz.com/adblocking/requests_top500.json.gz | gunzip > requests.json
 
-# VERSION: e1222d16430d754e8a2499fd8c58f49880d6c283
-./blockers/ublock:
-	git clone https://github.com/gorhill/ublock ./blockers/ublock
-	git clone --depth 1 https://github.com/uBlockOrigin/uAssets.git ./blockers/uAssets
-	cd ./blockers/ublock && git checkout --quiet e1222d16430d754e8a2499fd8c58f49880d6c283 && ./tools/make-nodejs.sh
+./blockers/ublock/.git:
+	git submodule update --recursive --depth 1 --init blockers/ublock
```
Comment (mjethani) [inline packages/adblocker-benchmarks/Makefile:8]: The `--recursive` option here ensures that uAssets is also fetched.

File: packages/adblocker-benchmarks/Makefile [hunk 3]
```diff
@@ -4,21 +4,19 @@ all: deps run
...
+
+./node_modules/uBO-snfe:
+	cd ./blockers/ublock && ./tools/make-nodejs.sh
 	npm install --no-save ./blockers/ublock/dist/build/uBlock0.nodejs
 
-ubo-snfe: ./blockers/ublock
+ubo-snfe: ./blockers/ublock/.git ./node_modules/uBO-snfe
```
Comment (mjethani) [inline packages/adblocker-benchmarks/Makefile:14]: The `./blockers/ublock` directory already exists now so we must refer to the `.git` file within the directory.

**Comment by mjethani** [comment]
@remusao everything is `--depth 1` now, and if the full history is needed one could always use `git fetch --unshallow` from within the submodule directory.

**Review by remusao — APPROVED** [review]

**Pull request merged**